.gitignore

```
@@ -111,4 +111,4 @@ temp/
 
 botError.png
 node_modules*/
-public/
+
```

Would be good to restore the `public` folder to the `.gitignore`, and also ignore these `.DS_Store` files

```suggestion
public/
.DS_Store
```